test/event_listeners.spec.js

```diff
@@ -170,6 +170,19 @@
         contentLength = function(body) {
           return sendRequest(body).httpRequest.headers['Content-Length'];
         };
+        it('ignores Content-Length for operations with an unsigned authtype', function(done) {
```

Comment:
Not a blocker, but the tests are much easier to read with empty lines separating them.